InnerEye/Common/metrics_dict.py

```diff
@@ -538,11 +547,6 @@ def get_pr_auc(self, hue: str = DEFAULT_HUE_KEY) -> float:
             return np.nan
         predictions = self.get_predictions(hue)
         labels = self.get_labels(hue)
-        if predictions.shape[1] == 1 and labels.shape[1] == 1 and len(np.unique(labels)) == 1:
```

Comment:
Why removing that?

Reply:
This is already the default behavior of  sklearn's precision_recall_curve() - there's a test case that makes sure of this as well.